command/agent/auth/kerberos/kerberos_test.go

```diff
@@ -57,11 +79,12 @@ func simpleAuthConfig() *auth.AuthConfig {
 		MountPath: "kerberos",
 		WrapTTL:   20,
 		Config: map[string]interface{}{
-			"username":      "grace",
-			"service":       "HTTP/05a65fad28ef.matrix.lan:8200",
-			"realm":         "MATRIX.LAN",
-			"keytab_path":   "grace.keytab",
-			"krb5conf_path": "krb5.conf",
+			"username":                 "grace",
+			"service":                  "HTTP/05a65fad28ef.matrix.lan:8200",
+			"realm":                    "MATRIX.LAN",
+			"keytab_path":              "grace.keytab",
+			"krb5conf_path":            "krb5.conf",
+			"disable_fast_negotiation": "true",
```

Comment:
Nit: this seems to be an optional parameter, so I'd think that `simpleAuthConfig` shouldn't have this by default. Maybe we could switch the test logic around, and first test for the absence of this param, and then test for it being set like so:

```
...
authConfig.Config["disable_fast_negotiation"] = "true"
authMethod, err = NewKerberosAuthMethod(authConfig)
...
```

Reply:
@calvn It is optional but the Kerberos auth method defines this as a bool, so it will always have a default of `false`: https://github.com/hashicorp/vault-plugin-auth-kerberos/blob/master/cli.go#L135.  So I was testing the override and then removing it to ensure it's false by default.  Thoughts?

Reply:
Yep, I was suggesting not setting this in `simpleAuthConfig` so that we can test on the default right away without having to `delete` the value.

If `simpleAuthConfig` didn't have `disable_fast_negotiation` set, we could do something like this for the test which should look a bit simpler:
```go
	authConfig = simpleAuthConfig()
	if authMethod, err := NewKerberosAuthMethod(authConfig); err != nil {
		t.Fatal("err shouldn't be returned for missing disable_fast_negotiation")
	}

	if authMethod.(*kerberosMethod).loginCfg.DisableFASTNegotiation != false {
		t.Fatalf("disable_fast_negotation should be false, it wasn't: %t", authMethod.(*kerberosMethod).loginCfg.DisableFASTNegotiation)
	}

	authConfig.Config["disable_fast_negotiation"] = "true"
	authMethod, err := NewKerberosAuthMethod(authConfig)
	if err != nil {
		t.Fatal(err)
	}

	if authMethod.(*kerberosMethod).loginCfg.DisableFASTNegotiation != true {
		t.Fatalf("disable_fast_negotation should be true, it wasn't: %t",
			authMethod.(*kerberosMethod).loginCfg.DisableFASTNegotiation)
	}
```

Reply:
Done!